From fa91cd33385154b38b18ca8eca8cb2d6d0714c32 Mon Sep 17 00:00:00 2001
From: Xiao Chen <xiao@cloudera.com>
Date: Thu, 20 Oct 2016 21:58:39 -0700
Subject: [PATCH 1957/2848] CLOUDERA-BUILD. CDH-35311.
 TestCrcCorruption#testCorruptionDuringWrt
 occasionally fails.

The test fails way more frequently on dist-test. Increase test timeout is proven to help.
See HDFS-6532.

Change-Id: Ib665f1ecd54fe166cd4a2d6ee9ed6a4e81b3963b
---
 .../org/apache/hadoop/hdfs/TestCrcCorruption.java  |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/TestCrcCorruption.java b/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/TestCrcCorruption.java
index 1d3be77..358956a 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/TestCrcCorruption.java
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/TestCrcCorruption.java
@@ -89,7 +89,7 @@ public void setUp() throws IOException {
    * create/write. To recover from corruption while writing, at
    * least two replicas are needed.
    */
-  @Test(timeout=50000)
+  @Test(timeout=90000)
   public void testCorruptionDuringWrt() throws Exception {
     Configuration conf = new HdfsConfiguration();
     // Set short retry timeouts so this test runs faster
-- 
1.7.9.5

